pyproject.toml

```diff
@@ -54,7 +54,7 @@ path = "install_scripts/hook.py"
 require-runtime-dependencies = true
 
 [tool.hatch.build.targets.wheel.force-include]
-"./pyccel/extensions/STC/include/stc" = "./pyccel/extensions/stc"
+"./pyccel/extensions/STC/include/stc" = "./pyccel/stc"
 
 [tool.hatch.build.targets.wheel.force_include_editable]
-"./pyccel/extensions/STC/include/stc" = "./pyccel/extensions/stc"
\ No newline at end of file
+"./pyccel/extensions/STC/include/stc" = "./pyccel/stc"
```

Comment:
Have you tried with `sdist` instead of `wheel`? I don't think a wheel is created for an editable installation

Reply:
Yes, unfortunately it didn't work. I'll need to look into it to identify the problem.